diff --git a/configure/cmake_win.cmake b/configure/cmake_win.cmake
index 9fdfe8d..2199c22 100644
--- a/configure/cmake_win.cmake
+++ b/configure/cmake_win.cmake
@@ -44,7 +44,7 @@ if(CMAKE_BUILD_TYPE STREQUAL "Debug")
                           "${dyn_defs}")
 
     target_compile_options(tangod PUBLIC ${ZMQ_PKG_CFLAGS_OTHER} ${OMNIORB_PKG_CFLAGS_OTHER} ${OMNICOS_PKG_CFLAGS_OTHER} ${OMNIDYN_PKG_CFLAGS_OTHER})
-    target_compile_options(tangod-static PUBLIC ${ZMQ_PKG_CFLAGS_OTHER} ${OMNIORB_PKG_CFLAGS_OTHER} ${OMNICOS_PKG_CFLAGS_OTHER} ${OMNIDYN_PKG_CFLAGS_OTHER} "/MTd")
+    target_compile_options(tangod-static PUBLIC ${ZMQ_PKG_CFLAGS_OTHER} ${OMNIORB_PKG_CFLAGS_OTHER} ${OMNICOS_PKG_CFLAGS_OTHER} ${OMNIDYN_PKG_CFLAGS_OTHER})
 
     target_link_libraries(tangod PUBLIC ${WIN32_LIBS} ${OMNIORB_PKG_LIBRARIES_DYN} ${ZMQ_PKG_LIBRARIES_DYN} ${PTHREAD_WIN_PKG_LIBRARIES_DYN} ${CMAKE_DL_LIBS})
     target_link_libraries(tangod-static PUBLIC ${WIN32_LIBS} ${OMNIORB_PKG_LIBRARIES_STA} ${ZMQ_PKG_LIBRARIES_STA} ${PTHREAD_WIN_PKG_LIBRARIES_STA} ${CMAKE_DL_LIBS})
@@ -101,7 +101,7 @@ else(CMAKE_BUILD_TYPE STREQUAL "Debug")
     set_target_properties(tango-static PROPERTIES PREFIX "lib")
  
     target_compile_options(tango PUBLIC ${ZMQ_PKG_CFLAGS_OTHER} ${OMNIORB_PKG_CFLAGS_OTHER} ${OMNICOS_PKG_CFLAGS_OTHER} ${OMNIDYN_PKG_CFLAGS_OTHER})
-    target_compile_options(tango-static PUBLIC ${ZMQ_PKG_CFLAGS_OTHER} ${OMNIORB_PKG_CFLAGS_OTHER} ${OMNICOS_PKG_CFLAGS_OTHER} ${OMNIDYN_PKG_CFLAGS_OTHER} "/MT")
+    target_compile_options(tango-static PUBLIC ${ZMQ_PKG_CFLAGS_OTHER} ${OMNIORB_PKG_CFLAGS_OTHER} ${OMNICOS_PKG_CFLAGS_OTHER} ${OMNIDYN_PKG_CFLAGS_OTHER})
 
     target_link_libraries(tango PUBLIC ${WIN32_LIBS} ${OMNIORB_PKG_LIBRARIES_DYN} ${ZMQ_PKG_LIBRARIES_DYN} ${PTHREAD_WIN_PKG_LIBRARIES_DYN} ${CMAKE_DL_LIBS})
     target_link_libraries(tango-static PUBLIC ${WIN32_LIBS} ${OMNIORB_PKG_LIBRARIES_STA} ${ZMQ_PKG_LIBRARIES_STA} ${PTHREAD_WIN_PKG_LIBRARIES_STA} ${CMAKE_DL_LIBS})
diff --git a/cppapi/client/CMakeLists.txt b/cppapi/client/CMakeLists.txt
index 5be3035..58e904f 100644
--- a/cppapi/client/CMakeLists.txt
+++ b/cppapi/client/CMakeLists.txt
@@ -63,11 +63,11 @@ if(WIN32)
     target_compile_definitions(client_objects_dyn PRIVATE "${dyn_defs}")
     target_compile_definitions(client_objects_sta PRIVATE "${static_defs}")
 
-    if(CMAKE_BUILD_TYPE STREQUAL "Debug")
-        target_compile_options(client_objects_sta PRIVATE "/MTd")
-    else()
-        target_compile_options(client_objects_sta PRIVATE "/MT")
-    endif()
+    # if(CMAKE_BUILD_TYPE STREQUAL "Debug")
+    #     target_compile_options(client_objects_sta PRIVATE "/MTd")
+    # else()
+    #     target_compile_options(client_objects_sta PRIVATE "/MT")
+    # endif()
 else(WIN32)
     add_library(client_objects OBJECT ${SOURCES})
     target_compile_options(client_objects PRIVATE -fPIC)
diff --git a/cppapi/server/CMakeLists.txt b/cppapi/server/CMakeLists.txt
index 1475915..00c4157 100644
--- a/cppapi/server/CMakeLists.txt
+++ b/cppapi/server/CMakeLists.txt
@@ -145,11 +145,11 @@ if(WIN32)
     target_compile_definitions(server_objects_dyn PRIVATE "${dyn_defs}")
     target_compile_definitions(server_objects_sta PRIVATE "${static_defs}")
 
-    if(CMAKE_BUILD_TYPE STREQUAL "Debug")
-        target_compile_options(server_objects_sta PRIVATE "/MTd")
-    else()
-        target_compile_options(server_objects_sta PRIVATE "/MT")
-    endif()
+    # if(CMAKE_BUILD_TYPE STREQUAL "Debug")
+    #     target_compile_options(server_objects_sta PRIVATE "/MTd")
+    # else()
+    #     target_compile_options(server_objects_sta PRIVATE "/MT")
+    # endif()
 else(WIN32)
     add_library(server_objects OBJECT ${SOURCES})
     target_compile_options(server_objects PRIVATE -fPIC)
diff --git a/cppapi/server/idl/CMakeLists.txt b/cppapi/server/idl/CMakeLists.txt
index cece00f..73bd44e 100644
--- a/cppapi/server/idl/CMakeLists.txt
+++ b/cppapi/server/idl/CMakeLists.txt
@@ -56,11 +56,11 @@ if(WIN32)
     target_compile_definitions(idl_objects_dyn PRIVATE "${dyn_defs};__x86__;__NT__;__OSVERSION__=4;__WIN32__;_WIN32_WINNT=0x0400;")
     target_compile_definitions(idl_objects_sta PRIVATE "${static_defs};__x86__;__NT__;__OSVERSION__=4;__WIN32__;_WIN32_WINNT=0x0400;")
 
-    if(CMAKE_BUILD_TYPE STREQUAL "Debug")
-        target_compile_options(idl_objects_sta PRIVATE "/MTd")
-    else()
-        target_compile_options(idl_objects_sta PRIVATE "/MT")
-    endif()
+    # if(CMAKE_BUILD_TYPE STREQUAL "Debug")
+    #     target_compile_options(idl_objects_sta PRIVATE "/MTd")
+    # else()
+    #     target_compile_options(idl_objects_sta PRIVATE "/MT")
+    # endif()
     install(FILES tango.h DESTINATION include/idl)
 else(WIN32)
     add_library(idl_objects OBJECT ${SOURCES} tango.h)
diff --git a/cppapi/server/jpeg/CMakeLists.txt b/cppapi/server/jpeg/CMakeLists.txt
index 08075c5..4a6a8f6 100644
--- a/cppapi/server/jpeg/CMakeLists.txt
+++ b/cppapi/server/jpeg/CMakeLists.txt
@@ -10,11 +10,11 @@ if(WIN32)
     target_compile_definitions(jpeg_objects_dyn PRIVATE "${dyn_defs}")
     target_compile_definitions(jpeg_objects_sta PRIVATE "${static_defs}")
 
-    if(CMAKE_BUILD_TYPE STREQUAL "Debug")
-        target_compile_options(jpeg_objects_sta PRIVATE "/MTd")
-    else()
-        target_compile_options(jpeg_objects_sta PRIVATE "/MT")
-    endif()
+    # if(CMAKE_BUILD_TYPE STREQUAL "Debug")
+    #     target_compile_options(jpeg_objects_sta PRIVATE "/MTd")
+    # else()
+    #     target_compile_options(jpeg_objects_sta PRIVATE "/MT")
+    # endif()
 else(WIN32)
     add_library(jpeg_objects OBJECT ${SOURCES})
     target_compile_options(jpeg_objects PRIVATE -fPIC)
diff --git a/cppapi/server/jpeg_mmx/CMakeLists.txt b/cppapi/server/jpeg_mmx/CMakeLists.txt
index 927e902..8f719d3 100644
--- a/cppapi/server/jpeg_mmx/CMakeLists.txt
+++ b/cppapi/server/jpeg_mmx/CMakeLists.txt
@@ -10,11 +10,11 @@ if(WIN32)
     target_compile_definitions(jpeg_mmx_objects_dyn PRIVATE "${dyn_defs}")
     target_compile_definitions(jpeg_mmx_objects_sta PRIVATE "${static_defs}")
 
-    if(CMAKE_BUILD_TYPE STREQUAL "Debug")
-        target_compile_options(jpeg_mmx_objects_sta PRIVATE "/MTd")
-    else()
-        target_compile_options(jpeg_mmx_objects_sta PRIVATE "/MT")
-    endif()
+    # if(CMAKE_BUILD_TYPE STREQUAL "Debug")
+    #     target_compile_options(jpeg_mmx_objects_sta PRIVATE "/MTd")
+    # else()
+    #     target_compile_options(jpeg_mmx_objects_sta PRIVATE "/MT")
+    # endif()
     if(${PLATFORM} EQUAL 64)
         target_compile_definitions(jpeg_mmx_objects_dyn PRIVATE _64BITS)
         target_compile_definitions(jpeg_mmx_objects_sta PRIVATE _64BITS)
diff --git a/log4tango/src/CMakeLists.txt b/log4tango/src/CMakeLists.txt
index a55b7ec..3d65650 100644
--- a/log4tango/src/CMakeLists.txt
+++ b/log4tango/src/CMakeLists.txt
@@ -30,11 +30,11 @@ if(WIN32)
     target_compile_definitions(log4tango_objects_dyn PRIVATE "${dyn_defs}")
     target_compile_definitions(log4tango_objects_sta PRIVATE "${static_defs}")
 
-    if(CMAKE_BUILD_TYPE STREQUAL "Debug")
-        target_compile_options(log4tango_objects_sta PRIVATE "/MTd")
-    else()
-        target_compile_options(log4tango_objects_sta PRIVATE "/MT")
-    endif()
+    # if(CMAKE_BUILD_TYPE STREQUAL "Debug")
+    #     target_compile_options(log4tango_objects_sta PRIVATE "/MTd")
+    # else()
+    #     target_compile_options(log4tango_objects_sta PRIVATE "/MT")
+    # endif()
 else(WIN32)
     add_library(log4tango_objects OBJECT ${SOURCES})
     target_compile_options(log4tango_objects PRIVATE -fPIC)
