diff --git a/configure/cmake_win.cmake b/configure/cmake_win.cmake
index 6f9f31c..9a49c7c 100644
--- a/configure/cmake_win.cmake
+++ b/configure/cmake_win.cmake
@@ -133,20 +133,18 @@ endif(CMAKE_BUILD_TYPE STREQUAL "Debug")
 if(CMAKE_BUILD_TYPE STREQUAL "Debug")
 
     install(TARGETS tangod-static
-            ARCHIVE DESTINATION lib COMPONENT static CONFIGURATIONS Debug
-            RUNTIME DESTINATION bin COMPONENT dynamic CONFIGURATIONS Debug)
+            ARCHIVE DESTINATION lib COMPONENT static CONFIGURATIONS Debug)
         
     install(TARGETS tangod
-            ARCHIVE DESTINATION lib COMPONENT static CONFIGURATIONS Debug
+            ARCHIVE DESTINATION lib COMPONENT dynamic CONFIGURATIONS Debug
             RUNTIME DESTINATION bin COMPONENT dynamic CONFIGURATIONS Debug)
 else(CMAKE_BUILD_TYPE STREQUAL "Debug")
 
     install(TARGETS tango-static
-            ARCHIVE DESTINATION lib COMPONENT static
-            RUNTIME DESTINATION bin COMPONENT dynamic)
+            ARCHIVE DESTINATION lib COMPONENT static)
         
     install(TARGETS tango
-            ARCHIVE DESTINATION lib COMPONENT static
+            ARCHIVE DESTINATION lib COMPONENT dynamic
             RUNTIME DESTINATION bin COMPONENT dynamic)
 endif(CMAKE_BUILD_TYPE STREQUAL "Debug")
 
