diff --git a/configure/cmake_win.cmake b/configure/cmake_win.cmake
index 9fdfe8d..8e6c2ad 100644
--- a/configure/cmake_win.cmake
+++ b/configure/cmake_win.cmake
@@ -17,7 +17,10 @@ file(WRITE ${PROJECT_BINARY_DIR}/VERSION ${LIBRARY_VERSION})
 include_directories(${INCLUDE_OBJECT_MS})
 include_directories(${PTHREAD_WIN_PKG_INCLUDE_DIRS})
 set(WIN32_LIBS "ws2_32.lib;mswsock.lib;advapi32.lib;comctl32.lib;odbc32.lib;")
-link_directories($ENV{PTHREAD_WIN}/lib)
+if(PTHREAD_WIN)
+    link_directories(${PTHREAD_WIN}/lib)
+endif(PTHREAD_WIN)
+
 if(CMAKE_BUILD_TYPE STREQUAL "Debug")
 
     add_library(tangod-static STATIC    $<TARGET_OBJECTS:log4tango_objects_sta>
@@ -310,19 +313,20 @@ if(CMAKE_VS_PLATFORM_TOOLSET STREQUAL "v141")
 endif(CMAKE_VS_PLATFORM_TOOLSET STREQUAL "v141")
 
 #pthreads
-install(FILES $ENV{PTHREAD_WIN}/lib/pthreadVC2.lib DESTINATION lib COMPONENT static)
-install(FILES $ENV{PTHREAD_WIN}/lib/pthreadVC2-s.lib DESTINATION lib COMPONENT static)
-install(FILES $ENV{PTHREAD_WIN}/bin/pthreadVC2.dll DESTINATION bin COMPONENT dynamic)
-install(FILES $ENV{PTHREAD_WIN}/bin/pthreadVC2.pdb DESTINATION bin COMPONENT dynamic)
-install(FILES $ENV{PTHREAD_WIN}/bin/pthreadVC2.exp DESTINATION bin COMPONENT dynamic)
-install(FILES $ENV{PTHREAD_WIN}/bin/pthreadVC2.ilk DESTINATION bin COMPONENT dynamic)
-install(FILES $ENV{PTHREAD_WIN}/lib/pthreadVC2d.lib DESTINATION lib COMPONENT static)
-install(FILES $ENV{PTHREAD_WIN}/lib/pthreadVC2-sd.lib DESTINATION lib COMPONENT static)
-install(FILES $ENV{PTHREAD_WIN}/bin/pthreadVC2d.dll DESTINATION bin COMPONENT dynamic)
-install(FILES $ENV{PTHREAD_WIN}/bin/pthreadVC2d.pdb DESTINATION bin COMPONENT dynamic)
-install(FILES $ENV{PTHREAD_WIN}/bin/pthreadVC2d.exp DESTINATION bin COMPONENT dynamic)
-install(FILES $ENV{PTHREAD_WIN}/bin/pthreadVC2d.ilk DESTINATION bin COMPONENT dynamic)
-
+if (PTHREAD_WIN)
+    install(FILES ${PTHREAD_WIN}/lib/pthreadVC2.lib DESTINATION lib COMPONENT static)
+    install(FILES ${PTHREAD_WIN}/lib/pthreadVC2-s.lib DESTINATION lib COMPONENT static)
+    install(FILES ${PTHREAD_WIN}/bin/pthreadVC2.dll DESTINATION bin COMPONENT dynamic)
+    install(FILES ${PTHREAD_WIN}/bin/pthreadVC2.pdb DESTINATION bin COMPONENT dynamic)
+    install(FILES ${PTHREAD_WIN}/bin/pthreadVC2.exp DESTINATION bin COMPONENT dynamic)
+    install(FILES ${PTHREAD_WIN}/bin/pthreadVC2.ilk DESTINATION bin COMPONENT dynamic)
+    install(FILES ${PTHREAD_WIN}/lib/pthreadVC2d.lib DESTINATION lib COMPONENT static)
+    install(FILES ${PTHREAD_WIN}/lib/pthreadVC2-sd.lib DESTINATION lib COMPONENT static)
+    install(FILES ${PTHREAD_WIN}/bin/pthreadVC2d.dll DESTINATION bin COMPONENT dynamic)
+    install(FILES ${PTHREAD_WIN}/bin/pthreadVC2d.pdb DESTINATION bin COMPONENT dynamic)
+    install(FILES ${PTHREAD_WIN}/bin/pthreadVC2d.exp DESTINATION bin COMPONENT dynamic)
+    install(FILES ${PTHREAD_WIN}/bin/pthreadVC2d.ilk DESTINATION bin COMPONENT dynamic)
+endif(PTHREAD_WIN)
 
 configure_file(tango.pc.cmake tango.pc @ONLY)
 
