diff --git a/cppapi/server/tango_config.h b/cppapi/server/tango_config.h
index 9c5a861..991559c 100644
--- a/cppapi/server/tango_config.h
+++ b/cppapi/server/tango_config.h
@@ -60,13 +60,21 @@
 //
 
 #ifdef _WIN32
-	#define		__WIN32__
-	#define		__x86__
-	#ifndef _WIN32_WINNT
-		#define		_WIN32_WINNT 0x0500
-	#endif
-	#define		__NT__
-	#define		__OSVERSION 4
+  #ifndef __WIN32__
+    #define __WIN32__
+  #endif
+  #ifndef __x86__
+    #define __x86__
+  #endif
+  #ifndef _WIN32_WINNT
+    #define _WIN32_WINNT 0x0500
+  #endif
+  #ifndef __NT__
+    #define __NT__
+  #endif
+  #ifndef __OSVERSION
+    #define __OSVERSION 4
+  #endif
 #endif
 
 //
